### ISSUE TITLE
[RTL] `layoutDirection: 'rtl'` does not work with HTML `dir=ltr` mode

### ISSUE THREAD

**Issue by budnix** [handsontable/handsontable#9226]
### Description

Setting the `layoutDirection: 'rtl'` option forces the table to be rendered from right to left and it works. But some of the UI elements are not positioned correctly. The issue is caused by incorrect Handsontable CSS selectors. The RTL selectors (`[dir-rtl] ...`) apply the rules only to the Handsontable parent's element, but in fact, they should be applied to the root element of the table.

As the result, some of the UIs can be misaligned, mispositioned, and not perfectly styled
![Kapture 2022-02-22 at 08 26 13](https://user-images.githubusercontent.com/571316/155083142-12e064c8-17dc-40b3-9945-ff3a57db62b0.gif)
(here, the manual column resize guide/ruler element is misplaced and the context menu has a too wide a border on the right)
 
The fix: All styles such as
```css
[dir=rtl] .htCommentCell:after {
  border-right: 6px solid transparent;
  border-left: none;
}
```
should be adjusted to:
```css
[dir=rtl].handsontable .htCommentCell:after {
  border-right: 6px solid transparent;
  border-left: none;
}
```

**Comment by aninde** [comment]
Fixed with v12.0.0 https://jsfiddle.net/38mL6run/

**Comment by AMBudnik** [comment]
Closed with https://github.com/handsontable/handsontable/issues/8760 (RTL Support), published in Handsontable v12.

For more information please check out our Release Notes - https://handsontable.com/docs/release-notes/#_12-0-0
